hooks/fetch/immatriculation-rne.ts

```diff
@@ -15,12 +14,6 @@ export function useFetchImmatriculationRNE(uniteLegale: IUniteLegale) {
           routes.api.rne.immatriculation + '/' + siren
         ),
       administration: EAdministration.INPI,
-      logError: (e: any) => {
-        logErrorInSentry(e, {
-          errorName: 'RNE API error',
-          siren,
-        });
-      },
```

Comment:
Shall we remove the error here or on server ? 

Does not it make it make the weird `failed to fecth` error silent ?

Reply:
I think I would rather remove the log on server.